gremlin-dotnet/src/Gremlin.Net/Process/Utils.cs

```diff
@@ -63,5 +66,23 @@ public static void Forget(this Task task)
                 t.Exception?.Handle(_ => true);
             }, TaskContinuationOptions.ExecuteSynchronously);
         }
+
+        /// <summary>
+        ///  Returns a user agent for connection request headers.
+        ///
+        /// Format:
+        /// "[Application Name] [GLV Name].[Version] [Language Runtime Version] [OS].[Version] [CPU Architecture]"
+        /// </summary>
+        private static string GenerateUserAgent()
+        {
+            var applicationName = Assembly.GetExecutingAssembly().GetName().Name.Replace(' ', '_');
+            var driverVersion = AssemblyName.GetAssemblyName("Gremlin.Net.dll")?.Version.ToString().Replace(' ', '_');
+            var languageVersion = Environment.Version.ToString().Replace(' ', '_');
+            var osName = Environment.OSVersion.Platform.ToString().Replace(' ', '_');
+            var osVersion = Environment.OSVersion.Version.ToString().Replace(' ', '_');
+            var cpuArchitecture = System.Runtime.InteropServices.RuntimeInformation.OSArchitecture.ToString().Replace(' ', '_');
```

Comment:
(nitpick) Please wrap this long line:
```suggestion
            var cpuArchitecture = System.Runtime.InteropServices.RuntimeInformation.OSArchitecture.ToString()
                .Replace(' ', '_');
```

or import this static classes so they don't need to be written out here with there namespaces.